src/Bookings.js

```diff
@@ -1,19 +1,39 @@
-import React from "react";
+/* eslint-disable indent*/
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
-
+import SearchResults from "./SearchResults.js";
+// import bookingData from "./data/fakeBookings.json";
 const Bookings = () => {
-  const search = searchVal => {
+  const [bookings, setBookings] = useState([]);
+
+  const search = (searchVal, event) => {
     console.info("TO DO!", searchVal);
+    event.preventDefault();
   };
-
+  const toLowerCase = customerName => {
+    return customerName.toLowerCase();
+  };
+  useEffect(searchVal => {
+    fetch(`https://cyf-react.glitch.me`)
+      .then(res => res.json())
+      .then(data => {
+        data = data;
+        setBookings(data => {
+          return data.filter(elem =>
+            [elem.firstName.toLowerCase(), elem.surname.toLowerCase()].includes(
+              searchVal.toLowerCase()
+            )
+          );
+        });
+      });
```

Comment:
This is a really interesting way of approaching this, combining the search and the fetch, it works well here! Just be cautious to ensure that it doesn't become too complicated, readability of your code should be always a priority